tests/infrastructure/instance_types/test_cpu_memory_hotplug_instancetype.py

```diff
@@ -119,15 +149,15 @@ def test_restart_hotplugged_vm(self, instance_type_hotplug_vm):
 
     @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME}::hotplug_cpu_instance_type"])
     @pytest.mark.polarion("CNV-11404")
-    def test_decrease_cpu_value(self, instance_type_hotplug_vm):
-        hotplug_instance_type_vm(vm=instance_type_hotplug_vm, sockets=FOUR_CPU_SOCKETS)
+    def test_decrease_cpu_value(self, instance_type_hotplug_vm, four_sockets_instance_type):
```

Comment:
should we have another test case dependent on "test_decrease_cpu_value" to check the cpu changes after we assert the restartrequired condition and restarted the Vm?

Reply:
This test checks for the error showing as you mentioned.
There is a tier 1 test to check for decreasing CPU values, so I see no value in creating it.

Reply:
do you mean we have this complete test with decreased cpu value upstream? if you are sure should we skip here ?
if we don't check the change after completely , having half test here would not be very useful.
probably maintainers have a opinion as well

Reply:
I appreciate the clarification. I understand your point now and will update the test accordingly